test/functionalTest/cases/2986_apiv2_notifications/apiv2_notification_reception.test

```diff
@@ -0,0 +1,382 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Reception of APIv2 notifications
+
+--SHELL-INIT--
```

Comment:
CB -> CB1
CP1 -> CB2
CP2 -> CB3

(dbInit, echo messages, etc.).

CP (Context Providers) are related with "NGSI9-based federation". For notification-based federeation, all CBs are equal citizens so better to adjust names.

Reply:
NTC